Project.toml

```
@@ -17,13 +17,13 @@ StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 [compat]
 CachedInterpolations = "0.1"
 CenterIndexedArrays = "0.1, 0.2"
-CoordinateTransformations = "0.5"
+CoordinateTransformations = "0.5, 0.6"
 Interpolations = "0.9, 0.10, 0.11, 0.12, 0.13"
-RegisterCore = "0.2.1"
+RegisterCore = "0.2.1, 0.2.2, 0.2.3, 0.2.4"
```

```suggestion
RegisterCore = "0.2.1"
```

This doesn't have to be changed because `RegisterCore = "0.2.1"` means "any 0.2.x version, as long as it's >= 0.2.1"